### PR TITLE
Linux sysctls configuration and Kubelet config

### PR DESCRIPTION
**Pull request by sergiotejon**

### CODE REVIEW

**Comment by juliocc** [comment]
Hola @sergiotejon! 

Thanks for the patch. LGTM but can you fix the linting issues (variables should be sorted alphabetically) and sign the CLA?

**Comment by juliocc** [comment]
Almost there. You have to run the tfdoc script (in the tools folder) in modules/gke-nodepool

**Comment by sergiotejon** [comment]
Gracias Julio :) .

Once you read CONTRIBUTING.md is when you know what you need to do :P.

File: modules/gke-nodepool/variables.tf [hunk 1]
```diff
@@ -40,6 +40,18 @@ variable "initial_node_count" {
   default     = 1
 }
 
+variable "kubelet_config" {
+  description = "Kubelet configuration."
+  type        = map(string)
```
Comment (ludoo) [inline modules/gke-nodepool/variables.tf:45]: Would it make sense to use an object type, to ensure there are no typos?

```
type = object({
  cpu_cfs_quota = string
  cpu_cfs_quota_period = string
  cpu_manager_policy = string
})
```

**Review by ludoo — APPROVED** [review]

File: modules/gke-nodepool/variables.tf (hunk 1 above)
Comment (juliocc) [reply modules/gke-nodepool/variables.tf:45]: Only those 3 are [supported](https://cloud.google.com/sdk/gcloud/reference/beta/container/node-pools/create#--system-config-from-file), so probably a good idea. And same thing for sysctls.

Comment (sergiotejon) [reply modules/gke-nodepool/variables.tf]: It makes all sense, of course. Uploaded fix.

**Pull request merged**